src/tree/hist/histogram.h

```diff
@@ -27,6 +28,7 @@
 #include "xgboost/logging.h"                    // for CHECK_GE
 #include "xgboost/span.h"                       // for Span
 #include "xgboost/tree_model.h"                 // for RegTree
+#include "../../processing/processor.h"         // for Processor
```

Comment:
Let's rename this into something more specific, `Processer` might not be clear to others.

Reply:
@nvidianz